pages/join.html

```diff
@@ -5,46 +5,29 @@
 home: false
 ---
 
-<h2>Join content goes here.</h2>
-
-<div class="contentContainer">
-  <div class="cardsContainer">
-    <div class="padded-content cards">
```

Comment:
One more thing. It looks like this section was from a [previous ticket](https://trello.com/c/VuBaiJvq/113-join-us-project-cadence) that goes above the section this PR references.

Reply:
I added .contentContainer as a parent div. I left .cardsContainer and .cards out bc it added duplicative padding. Padded-content is included.

Reply:
It still looks like the "project cadence" section is removed in this branch. 
<img width="394" alt="Screen Shot 2021-05-25 at 3 50 42 PM" src="https://user-images.githubusercontent.com/6326891/119566747-1cf98200-bd71-11eb-8c70-5789750d300d.png">

Reply:
Ok, I think I finally got it...added back project cadence